README.md

```diff
@@ -56,7 +56,7 @@ The [examples](examples) directory includes the following examples (and more).
 
 ## How Jib Works
```

Comment:
Do any of the jib-gradle, jib-maven or jib-core examples mention distroless?

Reply:
Did a `git grep -i distroless`, and no examples explicitly set distroless (although it's likely that many of them just don't set `from.image` at all).

Reply:
Nice!